src/Compilers/VisualBasic/Portable/Errors/Errors.vb

```diff
@@ -1957,7 +1957,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         WRN_Experimental = 42380
 
         ' // AVAILABLE                             42381 - 49998
-        ERRWRN_Last = WRN_Experimental
+        ERRWRN_NextAvailable = 42381
```

Comment:
It's only a single value and it's in the same file.
Maybe there is a simple unit test we could check for it 😏